### ISSUE TITLE
NTR: Aldosterone stays high in suppression tests (Endocrine & genitourinary)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7954]
New term request Aldosterone stays high in suppression tests (alexan_100126161036): Oral sodium loading, fludrocortisone, or captopril; or saline infusion

**Comment by pnrobinson** [comment]
@MickeySegal

Does this issue refer to a Dexamethasone Suppression Test?

If so would this be good
Blunted aldosterone response to dexamethasone suppression test
def: The dexamethasone suppression test (DST) involves the administration of dexamethasone orally for 2 days, followed by measuring plasma cortisol to demonstrate adequate suppression of adrenocorticotropic hormone (ACTH) and of aldosterone levels on day 3. This term applies to a DST result in which aldosterone levels fail to be suppressed (stay high).
PMID:11416708

Parent Abnormal response to endocrine stimulation test HP:0031073

**Comment by MickeySegal** [comment]
The testing for primary aldosteronism is more widespread, as detailed at https://ojrd.biomedcentral.com/articles/10.1186/1750-1172-5-9/tables/3

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000362
- label: Blunted aldosterone response to aldosterone suppression test
- parents: [Abnormal response to endocrine stimulation test (HP:0031073)](https://hpo.jax.org/app/browse/term/HP:0031073)
- pmid: PMID:20482833
- def: A non-suppressible aldosterone excess, defined as an abnormal result of one of any four suppression tests. These tests use oral sodium loading, oral fludrocortisone, oral captopril, or saline infusion to suppress aldosterone secretion.
- comment: For instance, the sral sodium loading test involves increasing sodium intake to over 200 mmol/d for 3 days and determining urinary aldosterone from the morning of day 3 to the morning of day 4. The test is abnormal if the amount of urinary aldosterone is above the upper limit of normal. An abnormal test result generally indicates primary aldosteronism.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7954
- synonyms:
